cloud/google/machineactuator.go

```diff
@@ -70,6 +72,14 @@ const (
 	noEventAction     = ""
 )
 
+func init() {
+	actuator, err := NewMachineActuator(MachineActuatorParams{})
+	if err != nil {
+		glog.Fatalf("error creating cluster provisioner for google : %v", err)
```

Comment:
nit: Fatals should use strings that start with a capital letter.

Reply:
Done